kano_profile/badges.py

```diff
@@ -69,6 +69,29 @@ def calculate_xp():
     return int(points) + qm.evaluate_xp()
 
 
+def calculate_app_progress(app_name):
+    def get_group_progress(group_profile):
```

Comment:
Only because it could be confusing externally - it might suggest that you give it the full profile and it will return the progress for every app's groups but it should be alright outside so I'll change it.